wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationMessageSnapshotTestCase.swift

```diff
@@ -47,6 +47,7 @@ func XCTAssertArrayEqual(
 class ConversationMessageSnapshotTestCase: ZMSnapshotTestCase {
 
     var userSession: UserSessionMock!
+    var forceRecord: Bool?
```

Comment:
question: what's the semantic meaning of the three `Bool?` values? `true`, `false`, `nil`